### PR TITLE
Paths with spaces fixes #70

### LINKED ISSUE #70: Paths with spaces are not commited
Status connected to ssh://git@example.com/XXX.git
Following remote branch origin/master. 

`"wp-content/themes/max v.1/header.php"       updated in work tree`
`"wp-content/themes/max v.1/single.php"       updated in work tree`

No commit has been done in the past 4 weeks.

### PR DESCRIPTION
**Pull request by rciorba**

### CODE REVIEW

File: tests/test-git-wrapper.php [hunk 1]
```diff
@@ -233,4 +235,62 @@ function test_git_dir_constant() {
...
+		$this->delete_on_teardown[] = $dir;
+		try {
+			mkdir( $dir, 0777, true );
+		} catch (Exception $_) { }
+		file_put_contents( $dir . "some file", "ana are mere" );
+		$this->assertTrue( $git->is_dirty() );
+	}
+
+	/**
+	 * Test is_dirty works for paths with whitespaces
```
Comment (bogdanpetrea) [inline tests/test-git-wrapper.php]: This is the same comment as above. Is this how it's supposed to be?

Comment (rciorba) [inline tests/test-git-wrapper.php]: copy-pasta fail.. will fix

File: gitium/inc/class-git-wrapper.php [hunk 2]
```diff
@@ -444,6 +447,12 @@ function get_local_changes() {
 				$y    = substr( $item, 1, 1 ); // Y shows the status of the work tree
 				$file = substr( $item, 3 );
 
+				if ( ( '"' == $file[0] ) && ('"' == $file[strlen( $file ) - 1] ) ) {
+					// git status --procelain will put quotes arround paths with whitespaces
```
Comment (bogdanpetrea) [inline gitium/inc/class-git-wrapper.php]: Some typos: --procelain and arround :D

**Comment by Marius786** [comment]
@rciorba LTM!

**Pull request merged**